chromium/util.js

```diff
@@ -14,7 +14,7 @@ var WARN = 5;
 // (Developers can just type DEFAULT_LOG_LEVEL=VERB in the console)
 var DEFAULT_LOG_LEVEL = NOTE;
 console.log("Hey developer! Want to see more verbose logging?");
-console.log("Type this into the console: DEFAULT_LOG_LEVEL=VERB");
+console.log("Type this into the console: require.scopes.util.setDefaultLogLevel(require.scopes.util.VERB);");
```

Comment:
`scopes` shouldn't be exposed to the user - this is for internal `require` usage.  I think it better to use the following code in the message:

```js
let util = require('./util'); util.setDefaultLogLevel(util.VERB);
```